.ci-builds/.azure-pipelines.yml

```diff
@@ -394,3 +394,59 @@ jobs:
       env:
         CC: clang
         CXX: clang++
+#-------------------------------------------------------------------------
+- job: ICU4C_Cygwin_GCC_x86_64_Release
+  displayName: 'C: Cygwin GCC x86_64 Release'
+  timeoutInMinutes: 60
+  pool:
+    vmImage: 'vs2017-win2016'
+  variables:
+    ICU_CI_CACHE: c:\icu-ci-cache
+    CYG_URL: https://cygwin.com/setup-x86_64.exe
+    CYG_MIRROR: http://mirrors.kernel.org/sourceware/cygwin/
+    CYG_PACKAGES: automake,gcc-core,gcc-g++,make,pkg-config,perl,python3
+    CYG_ROOT: c:\cygwin-root
+    CYG_CACHE: '$(ICU_CI_CACHE)\cygwin64-v3'
+    CYG_CACHED_SETUP: '$(CYG_CACHE)\setup.exe'
+  steps:
+    - script: |
+        git config --global core.autocrlf input
```

Comment:
I tried using `autocrlf false` but still end up getting errors for `\r` characters. I think there may be files that are already  checked-in using CRLF.

Reply:
Oh, good point -- I think you are right.
I think the `*.bat` files are checked-in using CRLF, for example: https://raw.githubusercontent.com/unicode-org/icu/main/icu4c/source/allinone/icucheck.bat has CRLF.

So perhaps this needs to stay as `input` then. I wonder if it might be worth adding a comment though...